woof-code/rootfs-skeleton/root/.xinitrc

```diff
@@ -153,19 +159,19 @@ if [ "$CURRENTWM" = "xfwm4" ];then
 fi
 
 #v3.91 volume tray applet, thanks to hairywill...
-[ -f /usr/bin/absvolume ] && absvolume &
+[ "$(which absvolume)" != "" ] && absvolume &
 
 #v3.95 support fbpanel tray/taskbar...
 #only launch tray for w.m. without inbuilt tray...
 if [ "$CURRENTWM" != "jwm" -a "$CURRENTWM" != "icewm" ];then
- if [ -f /usr/bin/fbpanel ];then
+ if [ "$(which fbpanel)" != "" ];then
   #a bit of a hack: when 3builddistro runs fixmenus, which calls variconlinks,
   #which populates /var/local/icons with symlinks, /usr/local/lib/X11/pixmaps
   #is not yet populated (happens at first boot, from default icon theme)...
   [ ! -e /var/local/icons/home48.png ] && ln -fs /usr/local/lib/X11/pixmaps/* /var/local/icons/
   fbpanel &
  fi
- [ "$desktop" = "rox" ] && [ -f /usr/bin/lxpanel ] && lxpanel &
+ [ "$desktop" = "rox" ] && [ "$(which lxpanel)" != "" ] && lxpanel &
```

Comment:
btw, you can use `command -v lpanel` instead of `which` - it should be faster because it's a shell builtin and not a separate process.

Reply:
Thanks for  the info. You're free to improve my pulled reuqest

Reply:
..and you can check the exit status instead of test and comparison = faster. I ~prefer~ have been using `type` (redirecting the output to /dev/null) but it is not POSIX and output as far as I know is the same between `type` and `command -v`